### DISCUSSION TITLE
How to limit/reduce detection.

### DISCUSSION THREAD

**Discussion by VagnPetersen** [blakeblackshear/frigate#10503] [Config Support]
### Describe the problem you are having

I have a simple setup with just one camera outside and a Coral USB detector. Running on an Odroid N2+. Everything works pretty well and I primarily use my setup to detect when my cats are leaving and coming back home.
I am using Home Assistant and I have some flows build in Node Red to send a message and snapshot through Telegram when a cat is detected. However I get too many messages when the cat is sitting outside in a zone and just waiting to be let in. How can I limit the detection "rate" or filtering a detection if the object has not moved since last detection?

### Version

0.13.2

### Frigate config file

```yaml
None
```

### Relevant log output

```shell
None
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

**Comment by NickM-27** [comment]
assuming you are referring to mqtt, you should use the available fields to only send a notification when you actually want one. For example, when stationary == false https://docs.frigate.video/integrations/mqtt#frigateevents

**Reply by VagnPetersen** [reply]
I am actually using the hass integration. From that one I have access to "Cat Occupancy" and "Cat count". Staying with just using the hass integration I guess I could play a little with "Motion" in combination with the two other properties mentioned.
If that's not working, I guess I have to look more into mqtt (I am using that too).
